vagrant/Vagrantfile

```diff
@@ -72,13 +72,30 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
             end		
           end		
         end
+        if ENV['PYTHON_REPOS']
+          config.vm.provision "file", source: "./rackhd-pm2-config.yml", destination: "~/rackhd-pm2-config.yml"
+          config.vm.provision "shell" do |s|		
+            s.inline = "apt-get install python-pip -y"		
+            s.privileged = true
+         end
+          list = ENV['PYTHON_REPOS'].split(' ')
+          list && list.each do |repo|
+            config.vm.provision "shell" do |s|		
+            s.inline = "cd /home/vagrant/src/#{repo} && pip install -r requirements.txt"		
```

Comment:
We still need to run pip install requirements here. And control the list of services from Jenkins job using PYTHON_REPOS Emc variable